app/views/posts/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with(model: @post, url: user_posts_path, class:"py-3") do |form| %>
+    <div class="field mb-3">
+        <%= form.label :post_title, class:"form-label" %>
+        <%= form.text_field :title, class:"form-control" %>
+    </div>
+
+    <div class="field, mb-3">
+        <%= form.label :post_text, class:"form-label" %>
+        <%= form.text_area :text, class:"form-control" %>
+    </div>
+
+    <div class="actions">
+        <%= form.submit class:"btn btn-primary" %>
+    </div>
```

Comment:
Thank you for your great suggestion!
a new post button is available on the posts index page. I think it is not an intuitive place; because the button is related to the current user, but current views show content that might not be considered owned by the current user. so I'll try to include it in the application layout to be on all pages